packages/babel-cli/test/index.js

```diff
@@ -174,7 +174,9 @@ fs.readdirSync(fixtureLoc).forEach(function (binName) {
         opts.inFiles[".babelrc"] = helper.readFile(babelrcLoc);
       }
 
-      it(testName, buildTest(binName, testName, opts));
+      if (testName !== "flag-dashed-inspect" || !process.version.match(/^v[045]/)) {
```

Comment:
We recently landed https://github.com/babel/babel/pull/5765, which allows you to specify a minimum node.js version via `minNodeVersion`